cmd/podman/root.go

```diff
@@ -208,7 +208,7 @@ func syslogHook() {
 
 func rootFlags(opts *entities.PodmanConfig, flags *pflag.FlagSet) {
 	// V2 flags
-	flags.StringVarP(&opts.Uri, "remote", "r", "", "URL to access Podman service")
+	flags.StringVarP(&opts.Uri, "remote", "r", registry.DefaultAPIAddress, "URL to access Podman service")
```

Comment:
Does this work on rootless and rootfull?  Or do we need a function call?

Reply:
Good point! Currently, `podman system service` does not distinguish between root and rootless. However, it seems like a follow up PR until rootless works.